lib/ui/hooks.dart

```diff
@@ -269,7 +269,7 @@ void _invoke(void callback(), Zone zone) {
 }
 
 /// Invokes [callback] inside the given [zone] passing it [arg].
-void _invoke1<A>(void callback(A a), Zone zone, A arg) {
+void _invoke1<A>(void callback(A a)/*?*/, Zone/*!*/ zone, A arg) {
```

Comment:
Thoughts on whether we should mark the callback params one way or another? Not entirely sure I trust the migration tool's judgement on the `_invoke*` calls.

Reply:
Good point. I think we should. These are basically anonymous `typedef`s, and I would definitely annotate a `typedef`.